unittest/test_util_Tokenizer.cpp

```diff
@@ -23,40 +23,51 @@
 TEST_CASE("util::Tokenizer")
 {
   using Mode = util::Tokenizer::Mode;
+  using IncludeDelimiter = util::Tokenizer::IncludeDelimiter;
+  const auto split_func = [](util::Tokenizer::Mode mode,
+                             IncludeDelimiter include_delimiter =
+                               IncludeDelimiter::no) {
+    return [mode, include_delimiter](nonstd::string_view string,
+                                     const char* separators) {
+      return Util::split_into_views(
+        string, separators, mode, include_delimiter);
+    };
+  };
 
   SUBCASE("include empty tokens")
   {
+    const auto split = split_func(Mode::include_empty);
     {
-      const auto s = Util::split_into_views("", "/", Mode::include_empty);
+      const auto s = split("", "/");
       REQUIRE(s.size() == 1);
```

Comment:
It's the same as here

Reply:
Right, but that's not the same input string.